frontend/src/pages/article/components/article-view/styles.module.scss

```diff
@@ -100,3 +102,279 @@
   line-height: 145%;
   word-break: break-word;
 }
+
+.presentationAuthorName {
+  display: none;
+  padding: 10px 33px 0;
+  font-size: 16px;
+}
+
+@media only screen and (max-width: breakpoints.$medium) {
+  .body:not(.hasCover) .textWrapper {
+    padding-top: 18px;
+  }
+
+  .buttonsWrapper > button {
+    width: 20px;
+    height: 20px;
+  }
+
+  .title {
+    font-size: 20px;
+  }
+
+  .text {
+    margin-top: 10px;
+    font-size: 16px;
+  }
+
+  .authorWrapper {
+    row-gap: 10px;
+    padding: 8px;
+    column-gap: 8px;
+  }
+
+  .authorName {
+    font-size: 14px;
+  }
+
+  .authorInfo {
+    font-size: 12px;
+  }
+
+  .authorInfoValue {
+    margin-right: 2px;
+    margin-left: 8px;
+  }
+
+  .authorInfo:last-child::before {
+    margin-left: 8px;
+    padding-right: 4px;
+  }
+
+  .articleInfoWrapper {
+    padding: 0 8px 8px;
+  }
+
+  .articleInfoListWrapper {
+    margin-bottom: 8px;
+    column-gap: 8px;
+  }
+
+  .articleInfoListWrapper > svg {
+    width: 15px;
+    height: 15px;
+  }
+
+  .articleInfoListWrapper + svg {
+    width: 15px;
+    height: 15px;
+  }
+
+  .articleInfoList {
+    font-size: 12px;
+    column-gap: 8px;
+  }
+
+  .articleInfoItem::before {
+    margin-right: 4px;
+  }
+
+  .articleGenre {
+    margin-left: 8px;
+    font-size: 12px;
+  }
+
+  .authorDetails.authorDetailsModal {
+    left: 15px;
+    width: 235px;
+  }
+}
+
+@media only screen and (min-width: breakpoints.$medium) and (max-width: breakpoints.$extra-large) {
+  .body:not(.hasCover) .textWrapper {
+    padding-top: 18px;
+  }
+
+  .presentationAuthorName {
+    font-size: 18px;
+  }
+
+  .title {
+    font-size: 30px;
+  }
+
+  .text {
+    margin-top: 18px;
+  }
+
+  .authorWrapper {
```

Comment:
Can't we have this breakpoint depended styles in article-details component? instead of duplicating all of them